common/src/main/java/com/abrenoch/hyperiongrabber/common/util/HyperionGrabberOptions.java

```diff
@@ -0,0 +1,66 @@
+package com.abrenoch.hyperiongrabber.common.util;
+
+import android.util.Log;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+
+public class HyperionGrabberOptions {
+    private static final boolean DEBUG = false;
+    private static final String TAG = "HyperionGrabberOptions";
+
+    private final int MINIMUM_IMAGE_PACKET_SIZE; // how many bytes the minimal acceptable image quality is
+    private final int FRAME_RATE;
+
+    public HyperionGrabberOptions(int horizontalLED, int verticalLED, int frameRate) {
+
+        // To determine the minimal acceptable image packet size we take the count of the width & height
+        // of the LED pixels (that the user is driving via their hyperion server) and multiply them
+        // together and then by 3 (1 for each color in RGB). This will give us the count of the bytes
+        // that the minimal acceptable quality should be equal to or greater than.
+        MINIMUM_IMAGE_PACKET_SIZE = horizontalLED * verticalLED * 3;
+        FRAME_RATE = frameRate;
+
+        if (DEBUG) {
+            Log.d(TAG, "Horizontal LED Count: " + String.valueOf(horizontalLED));
+            Log.d(TAG, "Vertical LED Count: " + String.valueOf(verticalLED));
+            Log.d(TAG, "Minimum Image Packet: " + String.valueOf(MINIMUM_IMAGE_PACKET_SIZE));
+        }
+    }
+
+    // just returns the frame rate
```

Comment:
Love the elaborate comments in this file, but when a comment starts with "just", you are probably overdoing it :-)

Reply:
That one did seem out of place :).

I'll get these comment blocks updated!